app/code/core/Mage/Checkout/Block/Onepage/Abstract.php

```diff
@@ -195,31 +185,6 @@ public function getRegionHtmlSelect($type)
         return $select->getHtml();
     }
 
-    /**
-     * @return bool|mixed
-     * @throws Mage_Core_Model_Store_Exception
-     */
-    public function getCountryOptions()
-    {
-        $options    = false;
-        $useCache   = Mage::app()->useCache('config');
-        if ($useCache) {
-            $cacheId    = 'DIRECTORY_COUNTRY_SELECT_STORE_' . Mage::app()->getStore()->getCode();
-            $cacheTags  = ['config'];
-            if ($optionsCache = Mage::app()->loadCache($cacheId)) {
-                $options = unserialize($optionsCache, ['allowed_classes' => false]);
-            }
-        }
-
-        if ($options == false) {
-            $options = $this->getCountryCollection()->toOptionArray();
-            if ($useCache) {
-                Mage::app()->saveCache(serialize($options), $cacheId, $cacheTags);
-            }
-        }
-        return $options;
-    }
-
```

Comment:
If we remove a public method I think we have to move the PR to the `next` branch

Reply:
Good point👍🏻 Didn't realize it was public because it seemed to be used only within that same class..
Alternative would be to leave it there (unused) and deprecated it?

Reply:
Vote to add `@deprecated` comment and suggest alternative method.

Reply:
ok for me too to deprecate it, maybe I'd do another PR on the next branch removing the method, if possible

Reply:
alright, I re-added the public method 👍